internal/daemonset/daemonset.go

```diff
@@ -310,3 +322,44 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 
 	return labels
 }
+
+func MakeLoadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	loadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(loadCommand, ra.Load...)
+	}
+
+	if a := spec.Args; a != nil && len(a.Load) > 0 {
+		loadCommand = append(loadCommand, a.Load...)
+	} else {
+		loadCommand = append(loadCommand, "-v")
+	}
+
+	if dirName := spec.DirName; dirName != "" {
+		loadCommand = append(loadCommand, "-d", dirName)
```

Comment:
Where are we setting the default value `/opt` if the user didn't specified any?

Reply:
[In the CRD](https://github.com/qbarrand/oot-operator/pull/127/files/d3a3896d24aef1bd487e2008dbb62e9dee21151c#diff-3047cc52e57eb78321dde3abdbad760dc862ba7d07b6130015c2346ee29e74c0R114).